CogNative/main.py

```diff
@@ -0,0 +1,59 @@
+import os
+import wave
+
+from models.RTVC.RTVC import RTVC
+
+# INITIALIZE RTVC
+v = RTVC("models/RTVC/saved_models/default")
+
+# SET INPUT AUDIO FILE PATH
+file_path = input("Enter input audio file path:\n")
+assert os.path.exists(file_path), "ERROR: Path not found."
+
+# ENTER TEXT
+text = input("Enter text for voice clone:\n")
+assert text[-1] == ".", "ERROR: Punctuation missing."
+
+# OUTPUT FILE PATH
+output_path = input("Enter output audio path:\n")
+if "/" in output_path:
+    output_temp = output_path.split("/")
+    output_temp.pop(-1)
+    output_temp = "/".join(output_temp)
+    assert os.path.exists(output_temp), "ERROR: Path not found."
+
+# ENCODE (sometimes this takes time, so it is after inputs)
+v.encode_voice(file_path)
+
+# SEPARATE TEXT BY PERIOD
+input_subs = text.split('. ')
+
+# JOIN <span> SENTENCES TO BE SYNTHESIZED TOGETHER
+span = 3
```

Comment:
Might want to shorten span and include nonstandard termination punctuation for splitting.

Reply:
Shortened span to 2. We can add non-standard termination punctuation as a separate issue.